CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ Keep it human-readable, your future self will thank you!
 - Enforce same binning for histograms comparing true data to predicted data
 - Fix: Inference checkpoints are now saved according the frequency settings defined in the config
 
+### Fixed
+
+- Fix `TypeError` raised when trying to JSON serialise `datetime.timedelta` object
```

Comment:
@gareth-j could you add  #43 to this line, to link to the PR please. This will be a new part of the style.

Reply:
Done.